Gemfile

```diff
@@ -16,5 +16,7 @@ group :development, :test do
   gem 'rubocop', require: false
 end
 
+gem 'overcommit'
```

Comment:
@GowthamK01 Can you please check if we can add this gem only to the `development` group?

Reply:
@gomathi-mallow Checked and added the overcommit gem inside the `development` group